src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -0,0 +1,35 @@
+<sofie-row class="mb-3">
+    <form [formGroup]="actionTriggerForm" class="left-container">
+        <div [formGroupName]="'data'">
+            <sofie-text-input [label]="labelLabel" [formControlName]="'label'"></sofie-text-input>
+            <sofie-keyboard-input [label]="shortcutLabel" [formControlName]="'keys'"></sofie-keyboard-input>
+            <sofie-keyboard-input [label]="physicalMappingLabel" [formControlName]="'mappedToKeys'" [helpText]="physicalMappingHelpText"></sofie-keyboard-input>
+            <sofie-select [label]="triggerOnLabel" [formControlName]="'triggerOn'" [options]="triggerOnOptions"></sofie-select>
+
+            <ng-container *ngIf="selectedAction" [ngTemplateOutlet]="actionArguments"></ng-container>
+
+            <ng-container *ngIf="doesSelectedActionHaveActionArguments()">
+                <ng-container [ngSwitch]="getSelectedActionArgumentType()">
+                    <sofie-text-input *ngSwitchCase="ActionArgumentSchemaType.STRING" [label]="selectedAction?.argument?.name!" [formControlName]="'actionArguments'" [helpText]="selectedAction?.argument?.description!"></sofie-text-input>
+                    <sofie-number-input *ngSwitchCase="ActionArgumentSchemaType.NUMBER" [label]="selectedAction?.argument?.name!" [formControlName]="'actionArguments'" [helpText]="selectedAction?.argument?.description!"></sofie-number-input>
+                </ng-container>
+            </ng-container>
+        </div>
+    </form>
+    <sofie-action-selector [selectedAction]="selectedAction" (actionSelected)="selectAction($event)" class="ps-5"></sofie-action-selector>
+</sofie-row>
+<sofie-button-group>
+    <sofie-button (click)="cancel()" i18n>global.close.button</sofie-button>
+    <sofie-spacer></sofie-spacer>
+    <sofie-button (click)="save()" [type]="'PRIMARY'" [disabled]="!actionTriggerForm.valid">Save</sofie-button>
```

Comment:
This comes down to personal preference.

Reply:
>  But in the case it doesn't, then there is also tiny overhead of evaluating the expression instead of having a static assignment

This is moving dangerously close into premature optimization.